README.md

```diff
@@ -0,0 +1,50 @@
+# hosts
+
+Allow user-defined hosts entries.
+
+`hosts` is a `LD_PRELOAD` library that overrides the fetching of hostnames in applications
+with functionality that fetches these entries from a user-defined hosts file instead of `/etc/hosts`.
+
+By default `hosts` fetches hosts entries from `~/.hosts`, but `HOSTS_FILE` can be used to specify a different file as well.
+
+## `HOSTALIASES`
+
+Some variants of libc have support for the `HOSTALIASES` environment variable.
+This is similar to using `HOSTS_FILE` in this library.
+However, the variant of glibc used on Debian and ArchLinux, amongst others, do not have this feature.
+
```

Comment:
Let's remove this whole chapter, I don't think we want to keep this up to date with regards to what's implemented elsewhere.